admin-tool/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2018"
 
 [dependencies]
 anyhow = "1"
-openssl = "0.10"
+openssl = "0.10.35"
```

Comment:
These should be split out to a different commit as they're an unrelated change

Reply:
In fact these changes are already upstream, I think you may need to rebase

Reply:
my commit contains only two file changes https://github.com/fedora-iot/fido-device-onboard-rs/commit/7393c5f3852605a63bf55a0b9995700657ccfd16 . CONTRIBUTING.md and TESTING.md . above two changes are earlier ,I guess.

Reply:
Please rebase your branch and re-push, that should resolve it